examples/simple_examples/molecular_dynamics.ipynb

```diff
@@ -7,7 +7,7 @@
    "outputs": [],
    "source": [
     "import warnings\n",
-    "from pylj import md, sample\n",
+    "from pylj import md, sample, pairwise\n",
```

Comment:
Why are you importing `pairwise` here?

Reply:
left in accidentally after tests, now removed